Automattic/fresh-data
# Examples: Move to packages/ and update.

This updates the examples to use the split out
@fresh-data/react-provider package, and places them
under the packages directory so lerna can link to them.

To Test:

0. `npm install`, `npm run bootstrap`

In both `examples/hello-world` and `examples/wp-rest-api`
1. `npm install`, `npm start`
2. Ensure example application works as expected.
